src/runtime_src/core/common/device.cpp

```diff
@@ -62,18 +62,77 @@ get_xclbin_uuid() const
 
 void
 device::
-register_axlf(const axlf* top)
+get_section_info(const char *buff, axlf_section_kind kind, 
+        const char *&sect_info, size_t *sect_size)
```

Comment:
I am not understand the need for this function?

Reply:
Removed this.. Not required now.